go.mod

```diff
@@ -3,8 +3,15 @@ module github.com/gutakk/go-google-scraper
 go 1.15
 
 require (
+	github.com/cosmtrek/air v1.21.2 // indirect
```

Comment:
Having an issue to set this up 

![Screen Shot 2563-11-23 at 13 17 04](https://user-images.githubusercontent.com/696529/99933977-29375f00-2d8f-11eb-9917-27c3bc0da9b0.png)

So I cannot run the app at the moment.

Reply:
please check this [comment](https://github.com/gutakk/go-google-scraper/pull/22#issuecomment-732070303)